gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXLibGit2Sharp.csproj

```diff
@@ -84,17 +85,33 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <AXProxy Include="Class.Global.axproxy">
```

Comment:
This lib should not basically throw errors and we catch it on the sysversioncontrol class :)